nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/MemberAddressPicker.java

```diff
@@ -107,6 +107,15 @@ public InetSocketAddress getPublicAddress() {
      */
     private void findAppropriateInterfaces() {
         
+        if (localConfig.getPublicAddress() != null && !localConfig.getPublicAddress().isEmpty()) {
+            String address[] = localConfig.getPublicAddress().split(":");
+            if (address.length > 1) {
+                publicAddress = new InetSocketAddress(address[0], Integer.parseInt(address[1]));
+            } else {
+                publicAddress = new InetSocketAddress(address[0], Integer.parseInt(config.getStartPort()));
+            }
+        }
+        
         //
```

Comment:
Probably needs to be removed

Reply:
Why's that?

Reply:
I am pointing to random "//" without any following comment.